### PR TITLE
backport #1878

### PR DESCRIPTION
**Pull request by Deflaimun**
Backport #1878 

Preview: 
https://deploy-preview-1907--redpanda-docs-preview.netlify.app/current/manage/security/encryption

### CODE REVIEW

**Review by Feediver1 — APPROVED** [review]
Thx Paulo--lgtm

File: modules/manage/pages/security/encryption.adoc [hunk 1]
```diff
@@ -14,232 +14,232 @@ For each Redpanda broker, specify the key file (`broker.key`), the certificate f
...
+----
 
-.`broker.cnf`
-[,ini]
+==== 1. Create CA configuration file (ca.cnf)
+
+Edit the `distinguished_name` section with your own organization details. For `default_md`, `sha256` is the minimum message digest level. The `subjectAltName` must be accurate for the broker's certificate.
+
+.ca.cnf
+[,bash]
```
Comment (JakeSCahill) [inline modules/manage/pages/security/encryption.adoc:35]:
```suggestion
[,ini]
```

File: modules/manage/pages/security/encryption.adoc [hunk 2]
```diff
@@ -14,232 +14,232 @@ For each Redpanda broker, specify the key file (`broker.key`), the certificate f
...
-==== 1. Create CA configuration file (ca.cnf)
+==== 1. Create broker configuration file (broker.cnf)
 
-Edit the `distinguished_name` section with your own organization details. For `default_md`, `sha256` is the minimum message digest level. The `subjectAltName` must be accurate for the broker's certificate.
+A subject alternative name (SAN) indicates all domain names and IP addresses secured by the certificate. Depending on the address the client uses to connect to Redpanda, you might need to create a CNF file for each broker to modify the `alt_names` section with organizational details. For production usage, edit `alt_names` with DNS resolutions and/or the IP addresses.
 
-.`ca.cnf`
-[,ini]
+.broker.cnf
+[,bash]
```
Comment (JakeSCahill) [inline modules/manage/pages/security/encryption.adoc:122]:
```suggestion
[,ini]
```

File: modules/manage/pages/security/encryption.adoc [hunk 3]
```diff
@@ -14,232 +14,232 @@ For each Redpanda broker, specify the key file (`broker.key`), the certificate f
...
 ----
 
-==== 2. Generate CA private key and public certificate
-
-Create a CA key to self-sign certificates:
+You could configure alternative names with a single version of `broker.key`/`broker.crt`,
+as long as you update the certificate for all brokers in the cluster any time you
+edit an entry. For example:
 
 [,bash]
```
Comment (JakeSCahill) [inline modules/manage/pages/security/encryption.adoc:148]:
```suggestion
[,ini]
```

File: modules/manage/pages/security/encryption.adoc [hunk 4]
```diff
@@ -14,232 +14,232 @@ For each Redpanda broker, specify the key file (`broker.key`), the certificate f
...
+DNS.1 = broker1.example.com
+DNS.2 = broker2.example.com
+DNS.3 = broker3.example.com
 ----
 
-Create a public certificate for the CA:
+Additionally, you can configure alternative names using the public or private
+IP addresses of all your brokers. For example:
 
 [,bash]
```
Comment (JakeSCahill) [inline modules/manage/pages/security/encryption.adoc:159]:
```suggestion
[,ini]
```

**Review by JakeSCahill — COMMENTED** [review]
We should use the `ini` label for those config snippets.

Also we have broken link formatting here: https://deploy-preview-1907--redpanda-docs-preview.netlify.app/current/manage/security/encryption/#configure-mtls

**Pull request merged**